index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Personal-Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+</head>
+<body>
+  <header class="header">
+    <nav class="top-nav">
+      <div class="logo">
+        <p>
+          my logo
+        </p>
+      </div>
+      <nav class="nav-items">
+        <div class="hamburger-menu">
+          <div></div>
+          <div></div>
+          <div></div>
+        </div>
+      </nav>
+    </nav>
+    <section class="text">
+      <header class="text-section">
+        <h1 class="head-section">
+          I&#8217;m Adeka Glad to see you!
+        </h1>
+
+        <p class="para-section">
+          I&#8217;m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don&#8217;t hestiate to contact me.
+        </p>
```

Comment:
- [ ] nice work. Please make your work responsive until the desktop screen size
![Screenshot (743)_LI](https://user-images.githubusercontent.com/65298048/212120028-0f660cbb-736a-49f0-abb8-e7b4c03d0154.jpg)